rero_ils/modules/items/api/circulation.py

```diff
@@ -1164,6 +1166,25 @@ def get_number_of_loans(self):
                 LoanState.ITEM_RETURNED,
             ]).source().count()
 
+    def get_number_of_loans_with_fees(self):
```

Comment:
is it possible to create a unit testing for this method?

Reply:
done